app/src/main/java/com/keylesspalace/tusky/components/announcements/AnnouncementAdapter.kt

```diff
@@ -73,6 +73,13 @@ class AnnouncementAdapter(
             return
         }
 
+        // hide button if announcement badge limit is already reached
+        if (item.reactions.size >= 8) {
+            addReactionChip.visibility = View.GONE
+        } else {
+            addReactionChip.visibility = View.VISIBLE
+        }
```

Comment:
We have a neat extension function for this, you can write it as `addReactionChip.visible(item.reactions.size < 8)`

Reply:
✅ adjusted

Reply:
You got the condition the wrong way

Reply:
sorry, stupid me :)